rules/linux/lnx_system_info_discovery.yml

```diff
@@ -0,0 +1,48 @@
+action: global
+title: System Information Discovery
+id: 42df45e7-e6e9-43b5-8f26-bec5b39cc239
+status: stable
+description: Detects system information discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1082/T1082.md
+falsepositives:
+    - Legitimate administration activities
+level: low
+tags:
+    - attack.discovery
+    - attack.t1082
+---
+logsource:
+    product: linux
+    categories: process_creation
+detection:
+    selection:
+        CommandLine|contains:
+          - 'uname'
+          - 'hostname'
+          - 'uptime'
+          - 'lspci'
+          - 'dmidecode'
+          - 'lscpu'
+          - 'lsmod'
```

Comment:
I believe all of these are binaries, so that would make use of ProcessName more suitable than CommandLine in this scenario (: Could you please adjust this section?